src/Sentry.Unity/JsonSentryUnityOptions.cs

```diff
@@ -0,0 +1,94 @@
+using System.Text.Json;
+using Sentry.Unity.Extensions;
+using UnityEngine;
+
+namespace Sentry.Unity
```

Comment:
We could move this to another namespace like `Sentry.Unity.Json` for example?

The motivation is to avoid exposing types that are not really "user facing" in the SDK root namespace where, for example, we have `SentryUnity` class.

An alternative is to make this stuff internal and use `InternalsVisibleTo` to Sentry.Unity.Editor.
But right now we're exposing a bunch of SDK internal bits that we don't expect to bump the major version if we change/break since they are not designed to be used by the users of the SDK